main.py

```diff
@@ -109,10 +140,21 @@ def __iter__(self):
         y = x + 1
         for _ in range(confug.gen_num_inherited - x - 1):
             print(x, " + ", y)
-            new_generation.append(
-                Creature(population[sorted_scores_indexes[x]].Kp, population[sorted_scores_indexes[y]].Ki))
-            new_generation.append(
-                Creature(population[sorted_scores_indexes[y]].Kp, population[sorted_scores_indexes[x]].Ki))
+            if (config.stability_verification):
```

Comment:
In this implementation, if a result of a crossover is not stable, we are slowly reducing kp and ki to make it fit to the equation (P+I)^2 < 4I. Is this ok? Maybe we should introduce a param that will let user pick wether we want to do that or just drop creature (like crossover_unstable_policy = [drop, reduce]).

Reply:
Could you please explain what is the background of this stability method?